From: Peymaneh Nejad <p.nejad@posteo.de>
Date: Mon, 30 Aug 2021 15:06:41 +0200
Subject: Add var Version

Forwarded: not-needed
---
 cmd/main.go | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/cmd/main.go b/cmd/main.go
index 8a9a289..58e9b02 100644
--- a/cmd/main.go
+++ b/cmd/main.go
@@ -37,6 +37,8 @@ import (
 	"go.uber.org/zap"
 )
 
+var version string
+
 func init() {
 	// set a fitting User-Agent for ACME requests
 	goModule := caddy.GoModule()
@@ -441,6 +443,9 @@ func printEnvironment() {
 func CaddyVersion() string {
 	goModule := caddy.GoModule()
 	ver := goModule.Version
+        if version != "" {
+                ver = version
+        }
 	if goModule.Sum != "" {
 		ver += " " + goModule.Sum
 	}
