From: Peymaneh <peymaneh@posteo.net>
Date: Mon, 2 Jan 2023 21:30:28 +0100
Subject: Add short description for caddy command

---
 cmd/cobra.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmd/cobra.go b/cmd/cobra.go
index d2a546c..c3f1831 100644
--- a/cmd/cobra.go
+++ b/cmd/cobra.go
@@ -5,7 +5,8 @@ import (
 )
 
 var rootCmd = &cobra.Command{
-	Use: "caddy",
+	Use:   "caddy",
+	Short: "An extensible server platform",
 	Long: `Caddy is an extensible server platform written in Go.
 
 At its core, Caddy merely manages configuration. Modules are plugged
