From: Peymaneh <peymaneh@posteo.net>
Date: Mon, 2 Jan 2023 21:30:28 +0100
Subject: Add short description for caddy command

---
 cmd/cobra.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/cmd/cobra.go
+++ b/cmd/cobra.go
@@ -14,6 +14,7 @@
 		bin = "caddy"
 	}
 
+	short := "An extensible server platform"
 	long := caddy.CustomLongDescription
 	if long == "" {
 		long = `Caddy is an extensible server platform written in Go.
@@ -101,6 +102,7 @@
 
 	return &cobra.Command{
 		Use:  bin,
+		Short: short,
 		Long: long,
 		Example: `  $ caddy run
   $ caddy run --config caddy.json
